static/include/app/apis/generated/data.md

```diff
@@ -194,14 +220,18 @@ You can also find your binary data under the **Images**, **Point clouds**, or **
 
 ```python {class="line-numbers linkable-line-numbers"}
 from viam.proto.app.data import BinaryID
+from viam.utils import create_filter
 
+my_filter = create_filter(component_name="camera-1", organization_ids=["<YOUR-ORG-ID>"])
 binary_metadata = await data_client.binary_data_by_filter(
```

Comment:
```suggestion
binary_metadata, count, last = await data_client.binary_data_by_filter(
```

Reply:
This does not work. It produces this error: 
```
  File "/Users/sierraguequierre/Desktop/data-client-qa.py", line 394, in main
    binary_metadata, count, last = await data_client.binary_data_by_filter(
                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: DataClient.binary_data_by_filter() got an unexpected keyword argument 'include_file_data
```
the type returned is a list. I don't know why the error is like that but the way I have it does work and does not produce that error. Should I file a bug/are you able to reproduce? @npentrel

Reply:
for context this is the extent of my code that produces that error: 
`    binary_metadata, count, last = await data_client.binary_data_by_filter(
      include_file_data=False
    )
`

Reply:
actually nvm just tested it again and it doesn't now. no idea why that was happening but can't repro. weird